### ISSUE TITLE
Write DB access points

### ISSUE THREAD

**Issue by aj8uppal** [david-fisher/320-F20-Track-IV#91]
ES6 syntax, `async`, `await`.

**Comment by hoon0422** [comment]
I thought #89 was written in ES6 syntax with `async` and `await`. Can you show any example to be corrected?

**Comment by aj8uppal** [comment]
@hoon0422 yup, put the issue up for verbosity, it is pending review until we can assert that we are finished writing them :)
